uint16[] id_list

float64[] current__mA
float64[] position__deg
float64[] rotation
